azext_iot/_help.py

```diff
@@ -1036,6 +1036,19 @@
         az iot edge set-modules --hub-name {iothub_name} --device-id {device_id} --content ../modules_content.json
 """
 
+helps[
+    "iot edge read-modules"
```

Comment:
I think we should change the command name to `export-modules` rather than read.

Reply:
good point, will update!